src/GraphQL/Resolver/DataObject.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+/**
+ * Pimcore
+ *
+ * This source file is available under two different licenses:
+ * - GNU General Public License version 3 (GPLv3)
+ * - Pimcore Enterprise License (PEL)
+ * Full copyright and license information is available in
+ * LICENSE.md which is distributed with this source code.
+ *
+ * @copyright  Copyright (c) Pimcore GmbH (http://www.pimcore.org)
+ * @license    http://www.pimcore.org/license     GPLv3 and PEL
+ */
+
+namespace Pimcore\Bundle\DataHubBundle\GraphQL\Resolver;
+
+use GraphQL\Type\Definition\ResolveInfo;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Service;
+use Pimcore\Bundle\DataHubBundle\GraphQL\Traits\ServiceTrait;
+
+class DataObject
+{
+    use ServiceTrait;
+
+    public function __construct(Service $graphQlService)
+    {
+        $this->setGraphQLService($graphQlService);
+    }
+
+    /**
+     * @param array            $value
```

Comment:
this is probably not an array (see how it is used in getById)

Reply:
done